src/main/scala/magellan/io/ShapeWritable.scala

```diff
@@ -37,6 +37,7 @@ private[magellan] class ShapeWritable(shapeType: Int) extends Writable {
     val h = shapeType match {
       case 0 => new NullShapeReader()
       case 1 => new PointReader()
+      case 3 => new PolyLineReader()
```

Comment:
can you also add tests for reading shapefiles in polyline format? there is a polyline test file you can use for unit testing.
the main branch has a test that you can replicate